### ISSUE TITLE
The created job does not appear in the assigned job slice？

### ISSUE THREAD

**Issue by wtkiku** [vipshop/Saturn#236]
In the Executor overview list, I created some scripts that did not appear in the "assigned job slices", and when I manually executed the scripts was not responding, what conditions were in the limit?

![qq 20170908154516](https://user-images.githubusercontent.com/12605333/30201273-be220338-94ac-11e7-887e-31ac7db84efe.png)

**Comment by xiaoxing598** [comment]
The same problem happened to me here， In what circumstances the job created is not added to the list, hope at least one log prompt, tell me how to deal with! @hebelala

**Comment by hebelala** [comment]
The probable reason is，the executer failed to initialize the job，so can not take over it.  You can check the executer s log.

**Comment by xiaoxing598** [comment]
I only see a warning log, in the implementation of the task when the job, it can not provide me with enough information.

```
[2017-09-11 10:13:23.379] [WARN] [VM_231_32_centos-$Jobs-watcher] [com.vip.saturn.job.executor.SaturnExecutorService] >>> the job wxdc_create_day_task is unnecessary to initialize, because it's already existing
```

**Comment by hebelala** [comment]
The warn log have no problem. Are you sure the job is enabled, and it has been executed at least once?

**Comment by xiaoxing598** [comment]
Yes, in addition to configuring the timed implementation of the timed strategy, I also clicked "立即执行一次", but it did not perform successfully.

I am doing the upgrade work, these scripts copied from the ZK over, it should not be the problem of this operation, because I re-create the task, but still can not be final implementation, it is not assigned to the implementation of this matter can see.

**Comment by hebelala** [comment]
The job is already initialized.  The problems may be as follows:  
1. NamespaceShardingService occurs some problems. You can check the saturn-console logs;
2. The executors haven't a leader, or the leader doesn't work.  So, the shards cannot be allocated. You can check the executor logs.

**Comment by xiaoxing598** [comment]
I do not see a warning or error level log from "console" and "executor", and can provide enough log information for fragmentation issues that I can not target.
I want to know all the reasons why fragmentation can not be allocated, otherwise I can not fix it.

**Comment by hebelala** [comment]
Show me the saturn-console's log, when add new job.

**Comment by wtkiku** [comment]
When adding a new job, the saturn-console's log is displayed as follows:

`[2017-09-12 13:51:42.001] [INFO] [http-nio-9088-exec-2] [c.v.s.j.c.r.zookeeper.impl.CuratorRepositoryImpl] >>> job node value is null, node:/$Jobs/test2/config/customContext
[2017-09-12 13:51:42.080] [INFO] [http-nio-9088-exec-2] [c.v.s.j.c.r.zookeeper.impl.CuratorRepositoryImpl] >>> job node value is null, node:/$Jobs/test2/config/queueName
[2017-09-12 13:51:42.100] [INFO] [http-nio-9088-exec-2] [c.v.s.j.c.r.zookeeper.impl.CuratorRepositoryImpl] >>> job node value is null, node:/$Jobs/test2/config/channelName
[2017-09-12 13:52:12.348] [INFO] [get-systemConfig-to-memory-timer-1505185932342] [c.v.s.j.c.service.impl.SystemConfigServiceImpl] >>> get system config from db`

**Comment by hebelala** [comment]
I want the logs about `NamespaceShardingService`

**Comment by wtkiku** [comment]
sorry, I can't see any logs about ` NamespaceShardingService ` from the recent logs.
But I find a log from saturn-executor-error.log, is it helpful? 

> [2017-09-12 13:51:45.698] [WARN] [Curator-Framework-0] [org.apache.curator.utils.ZKPaths] >>> The version of ZooKeeper being used doesn't support Container nodes. CreateMode.PERSISTENT will be used instead.

**Comment by hebelala** [comment]
It's not helpful. Are you using the version 2.1.2?  The v2.1.2 wiki is https://github.com/vipshop/Saturn/wiki/Saturn2.1.2%E7%89%88%E6%9C%AC%E5%8D%87%E7%BA%A7%E6%8C%87%E5%8D%97  
Maybe you forget to configure `CONSOLE_ZK_CLUSTER_MAPPING`, the `NamespaceShardingService`  is not started.

**Comment by wtkiku** [comment]
I am using the version 2.1.2,
and I have configured `CONSOLE_ZK_CLUSTER_MAPPING` .
Some of the old job that I copied from the old version can work normally,but when I try to add a new job,it has problem as shown above.

**Comment by xiaoxing598** [comment]
If we are configured correctly, it should be like this.

CONSOLE_ZK_CLUSTER_MAPPING | console_VM_xxx_32_centos:/domain/dev.domain,/domain/test.domain,/domain/prod.domain
-- | --
